test/demo-logger-using-pvc/logger.py

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/python3
+
+import logging
+import socket
+import time
+import sys
+
+LOG_DIR = "/var/log/demoLogger/"
```

Comment:
log dir will be configurable as part of solution config, can read this from a sample config file
Keep a note here that Log dir will be read by CORX services from ConfStore

Reply:
Yes, but for demo purposes, I used constant variables only since, if I add the YAML config file etc, then pkg dependencies like cortx-py-utils or some third party pkg will arise, which will make the demo unnecessarily complicated.

If still needed, we can add that. (pls suggest whether to use cortx-py-utils or some external pkg to parse config in such case.)

Reply:
Just add a comment that CORTX services would reading this LOG DIR from Config

Reply:
Added the comment 👍